tests/Mvc/Database/DataImporterUnitTest.php

```
@@ -0,0 +1,634 @@
+		$mockAdapter = $this->createMock( AdapterInterface::class );
+		$mockConfig = $this->createMockConfig();
+
+		// Configure adapter mock
+		$mockAdapter->method( 'connect' );
+		$mockAdapter->method( 'getAdapterType' )->willReturn( 'mysql' );
+
+		// Mock AdapterFactory
+		$this->mockAdapterFactory( $mockAdapter );
+
+		// Test with progress callback
+		$called = false;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$called'. (undefined)

(UnusedLocalVariable)